Packs/AzureSecurityCenter/Integrations/MicrosoftDefenderForCloudEventCollector/MicrosoftDefenderForCloudEventCollector.py

```diff
@@ -0,0 +1,293 @@
+import demistomock as demisto
+from CommonServerPython import *
+import urllib3
+from MicrosoftApiModule import *  # noqa: E402
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+VENDOR = 'microsoft'
+PRODUCT = 'defender_for_cloud'
+API_VERSION = '2022-01-01'
```

Comment:
Will this never  change for any client ?

Reply:
What do you mean by this? This is the latest API-version that we want to use.